src/Microsoft.XmlSerializer.Generator/src/Microsoft.XmlSerializer.Generator.csproj

```diff
@@ -105,57 +81,18 @@
     <Compile Include="$(SourceDir)\System\Xml\Serialization\Types.cs">
       <Link>System\Xml\Serialization\Types.cs</Link>
     </Compile>
-    <Compile Include="$(SourceDir)\System\Xml\Serialization\XmlAnyAttributeAttribute.cs">
-      <Link>System\Xml\Serialization\XmlAnyAttributeAttribute.cs</Link>
```

Comment:
Can you please remove the `#ifdef` from the files that are excluded? For example, `XmlAnyAttributeAttribute.cs` still has the following code

```
 #if XMLSERIALIZERGENERATOR
 namespace Microsoft.XmlSerializer.Generator
 #else
 namespace System.Xml.Serialization
 #endif
```